app/models/vm_or_template/scanning.rb

```diff
@@ -36,6 +36,23 @@ def raw_scan(userid = "system", options = {})
     raise
   end
 
+  #
+  # Default Adjustment Multiplier is 1 (i.e. no change to timeout)
+  #   since this is a multiplier, timeout * 1 = timeout
+  #
+  # Subclasses MAY choose to override this
+  #
+  def self.scan_timeout_adjustment_multiplier
```

Comment:
Looks like this needs to be in a `module ClassMethods` for this to work from a mixin:

```
>> VmOrTemplate.scan_timeout_adjustment_multiplier
NoMethodError (undefined method `scan_timeout_adjustment_multiplier' for #<Class:0x0000556956a42238>)
```

If I change to
```
--- a/app/models/vm_or_template/scanning.rb
+++ b/app/models/vm_or_template/scanning.rb
@@ -42,8 +42,10 @@ module VmOrTemplate::Scanning
   #
   # Subclasses MAY choose to override this
   #
-  def self.scan_timeout_adjustment_multiplier
-    1
+  module ClassMethods
+    def scan_timeout_adjustment_multiplier
+      1
+    end
```

Then
```
>> VmOrTemplate.scan_timeout_adjustment_multiplier
=> 1
```

Reply:
Thanks - updated!